docs/enterprise-guide/authenticating-with-saml.md

```diff
@@ -94,93 +77,102 @@ contain attributes for each User's first name, last name, and email. The asserti
 </saml2:Assertion>
 ```
 
-Most SAML identity providers we've used already include these
-assertions by default, but some (such as Okta) must be configured to
-include them. Here's an example of what your assertions configuration
-should look like in Okta. (You can find this page by going to `Admin > Applications > Metabase > General > SAML Settings [Edit]`).
+Most SAML identity providers we've used already include these assertions by default, but some (such as Okta) must be configured to include them. Here's an example of what your assertions configuration should look like in Okta. (You can find this page by going to `Admin > Applications > Metabase > General > SAML Settings [Edit]`).
```

Comment:
This is an aside, as it affects just about all MB docs I've seen, but IMHO using back-ticks for non-code ("technical terms") text, is a misuse. We can discuss if you want :)